src/Network/Receive/ServerType0.pm

```diff
@@ -1153,14 +1162,16 @@ sub map_loaded {
 		main::initMapChangeVars();
 	} else {
 
-		$messageSender->sendSync(1) if ($masterServer->{serverType} eq 'bRO'); # tested at bRO 2013.11.26 - revok
+		$messageSender->sendMapConnected if $messageSender->can( 'sendMapConnected' );
```

Comment:
i think the if condition already checks whether the servertype can perform it or not